<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dMenu</class>
 <widget class="QWidget" name="dMenu">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Open Sans</family>
   </font>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="windowTitle">
   <string>МЕНЮ</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(238, 243, 250);</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="left">
     <property name="minimumSize">
      <size>
       <width>193</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>228</width>
       <height>10000000</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(14, 210, 226, 255), stop:0.885000 rgba(14, 210, 226, 255), stop:0.885001 rgba(0, 0, 0, 40), stop:0.980000 rgba(0, 0, 0, 0), stop:1 rgba(0, 0, 0, 0))</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="name">
        <property name="minimumSize">
         <size>
          <width>17</width>
          <height>86</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>171</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Open Sans</family>
          <pointsize>18</pointsize>
          <kerning>true</kerning>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0);
border-bottom:10px solid qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 0, 0, 32), stop:1 rgba(255, 255, 255, 0));</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;MULTIGRAPH&lt;br/&gt;Ver. 1.42&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>478</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QFrame" name="placeLogo">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>74</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0);</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <widget class="QWidget" name="shadow" native="true">
         <property name="geometry">
          <rect>
           <x>52</x>
           <y>7</y>
           <width>140</width>
           <height>50</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgba(0, 0, 0, 100);
border-right:15px solid qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 100), stop:1 rgba(255, 255, 255, 0));
border-bottom:15px solid qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 0, 0, 100), stop:1 rgba(255, 255, 255, 0));
border-left:15px solid qlineargradient(spread:pad, x1:1, y1:0, x2:0, y2:0, stop:0 rgba(0, 0, 0, 100), stop:1 rgba(255, 255, 255, 0));
border-top:15px solid qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgba(0, 0, 0, 100), stop:1 rgba(255, 255, 255, 0));
background-clip:padding;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>45</x>
           <y>0</y>
           <width>140</width>
           <height>50</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>C:/Work/logotpchel.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="date_time">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>109</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>171</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Open Sans</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 255, 255, 0);
border-bottom:10px solid qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(0, 0, 0, 32), stop:1 rgba(255, 255, 255, 0));
border-top:10px solid qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:1 rgba(0, 0, 0, 32), stop:0 rgba(255, 255, 255, 0));</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:22pt; color:#ffffff;&quot;&gt;08:40:12&lt;/span&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-size:17pt; color:#ffffff;&quot;&gt;12.12.2017&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="right">
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="main_frame">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QStackedWidget" name="stackedWidget">
           <property name="baseSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::TabFocus</enum>
           </property>
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="mainMenu">
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>11</number>
             </property>
             <property name="leftMargin">
              <number>37</number>
             </property>
             <property name="topMargin">
              <number>76</number>
             </property>
             <property name="rightMargin">
              <number>53</number>
             </property>
             <item>
              <widget class="wButtonStyled" name="bWork">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>736</width>
                 <height>83</height>
                </size>
               </property>
               <property name="focusPolicy">
                <enum>Qt::StrongFocus</enum>
               </property>
               <property name="text">
                <string>РАБОТА</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="wButtonStyled" name="bSettings">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>736</width>
                 <height>83</height>
                </size>
               </property>
               <property name="text">
                <string>НАСТРОЙКА</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="wButtonStyled" name="bDiagnost">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>736</width>
                 <height>83</height>
                </size>
               </property>
               <property name="text">
                <string>ДИАГНОСТИКА</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="wButtonStyled" name="bExpert">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>736</width>
                 <height>83</height>
                </size>
               </property>
               <property name="text">
                <string>ЭКСПЕРТ</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>227</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="work"/>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_buttons">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>180</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>182</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>25</number>
         </property>
         <property name="topMargin">
          <number>64</number>
         </property>
         <property name="rightMargin">
          <number>53</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>250</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="wButtonStyled" name="saveButton">
           <property name="minimumSize">
            <size>
             <width>233</width>
             <height>73</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>233</width>
             <height>73</height>
            </size>
           </property>
           <property name="text">
            <string>СОХРАНИТЬ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="wButtonStyled" name="exitButton">
           <property name="minimumSize">
            <size>
             <width>233</width>
             <height>73</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>233</width>
             <height>73</height>
            </size>
           </property>
           <property name="text">
            <string>ВЫХОД</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>wButtonStyled</class>
   <extends>QPushButton</extends>
   <header location="global">buttonstyled.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
